plugins/pencilblue/controllers/section.js

```diff
@@ -120,7 +120,7 @@ module.exports = function(pb) {
                 order: [{'publish_date': pb.DAO.DESC}, {'created': pb.DAO.DESC}]
             };
             pb.ArticleServiceV2.setPublishedClause(opts.where);
-            self.service.getBySectionViewController(section, opts, function(err, content) {
+            self.service.getBySection(section, opts, function(err, content) {
```

Comment:
This was throwing an exception because getBySectionViewController() doesn't exist.